modules/before-updating-ocp.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-web-console.adoc
+
+:_content-type: CONCEPT
+[id="before-updating-ocp{context}"]
+= Before updating the {product-title}
```

Comment:
This phrasing is probably correct in terms of grammar and technical accuracy, but I figure adding cluster to the end would sound better/more in line with our conventions of referring to cluster updates:
```suggestion
= Before updating the {product-title} cluster
```